content/docs/azure/deploy/uninstall-kubeflow.md

```diff
@@ -0,0 +1,15 @@
++++
+title = "Uninstall Kubeflow"
+description = "Instructions for uninstalling Kubeflow"
+weight = 10
++++
+
+## Uninstall Kubeflow on your Azure AKS cluster.
```

Comment:
Recommendation: change this from a heading to straight text. The page doesn't need a heading, as it's so short and the heading is equivalent to the page title.

Reply:
Apologies! I'll make some changes and send a PR right away.